packages/application/src/shell.ts

```diff
@@ -1303,6 +1303,24 @@ export class LabShell extends Widget implements JupyterFrontEnd.IShell {
     return layout;
   }
 
+  /**
+   * Toggle the fullscreen mode based on user inputs
+   */
+  toggleFullscreenMode(): void {
```

Comment:
Can I ask for one more thing - to move this method into a function in `packages/application-extension/src/index.tsx`?

The rationale is a follows: once we add something to `LabShell` class , any attempt to change it will be potentially breaking for an extension. Since there is further expansion of the fullscreen capabilities requested (to cover making individual elements take the full screen) enhancements, I think it is likely that the implementation may change.

Reply:
@krassowski we’ve removed the method and added it to `packages/application-extension/src/index.tsx`, let us know if it looks good